packages/@sanity/base/src/presence/overlay/WithIntersection.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import {tap} from 'rxjs/operators'
+import {ObservableIntersectionObserver} from './intersectionObserver'
```

Comment:
this would preferably be a type import: `import type {ObservableIntersectionObserver} from './intersectionObserver'`

Reply:
I will change it! :D 
Can you go into more detail as to what's the difference between one or the other so that I can tell when to use one vs the other next time? 🙏

Reply:
the type import will be removed entirely in transpiled code/at runtime. When importing types we should always use type imports, but we haven't gotten around to enforce it with tooling yet :)

Reply:
Gotcha 🙌 thanks!